src/app/features/manage-tokens/manage-tokens.tsx

```diff
@@ -0,0 +1,97 @@
+import { type ChangeEvent, useState } from 'react';
+import { useDispatch } from 'react-redux';
+
+import { Stack, VStack } from 'leather-styles/jsx';
+
+import { Button, Dialog } from '@leather.io/ui';
+
+import { isTokenEnabled } from '@app/common/filter-tokens';
+import { NoTokensFound } from '@app/components/no-tokens-found/no-tokens-found';
+import { BitcoinStandardsList } from '@app/features/asset-list/bitcoin/bitcoin-standards-list/bitcoin-standards-list';
+import { StacksAssetsList } from '@app/features/asset-list/stacks/stacks-assets-list/stacks-assets-list';
+import { useCurrentAccountIndex } from '@app/store/accounts/account';
+import { setToken, useAllTokens } from '@app/store/manage-tokens/manage-tokens.slice';
+import { Footer } from '@app/ui/components/containers/footers/footer';
+import { DialogHeader } from '@app/ui/components/containers/headers/dialog-header';
+import Toggle from '@app/ui/components/toggle/toggle';
+
+interface ManageTokensProps {
+  onClose(): void;
+}
+
+export type TokenProtocol = 'stacks' | 'bitcoin';
+
+export function ManageTokensDialog({ onClose }: ManageTokensProps) {
+  const [hasTokens, setHasTokens] = useState(false);
+  const accountIndex = useCurrentAccountIndex();
+  const allTokens = useAllTokens();
+  const dispatch = useDispatch();
+
+  const variant = 'read-only';
+  const isReadOnly = true;
+
+  const handleSelection = (e: ChangeEvent<HTMLInputElement>, isSelected: boolean) => {
```

Comment:
We have a convention of using `function` rather than `const` to be more declarative